docs/src/guide/tutorial.jl

```diff
@@ -45,10 +45,6 @@ basis = PlaneWaveBasis(model, Ecut; kgrid=kgrid)
 ## 3. Run the SCF procedure to obtain the ground state
 scfres = self_consistent_field(basis, tol=1e-8);
 
-# Note that DFTK by default applies the convergence tolerance `tol`
-# to the energy difference, so that the norm in the density difference
-# is not yet converged to 8 digits.
-#
```

Comment:
Why do you want to remove that?

Reply:
Because it's clearer now that there is the difference in energy why this stops

Reply:
true. :+1: